azext_edge/tests/utility/test_file_operations_unit.py

```diff
@@ -86,6 +153,7 @@ class TestScenario(NamedTuple):
         is_binary: bool = False
 
         def write_content(self) -> Union[bytes, str]:
+            file_name = os.path.abspath(os.path.expanduser(self.file_name))
```

Comment:
but muh ~

Reply:
I know, we either have a creative alternative or maybe not the worst thing since we aren't generating random file names, so manual cleanup (in case auto cleanup doesn't run for whatever reason) isn't that bad.